src/main/resources/api/swagger.yaml

```diff
@@ -225,9 +173,15 @@ definitions:
       author:
         type: string
         description: Name of the feature author
+      status:
+        type: string
+        description: Status of the feature.
       updated_at:
         type: string
         description: Last update date in ISO 8601 format
```

Comment:
This is the preferred way according to the api guide lines: http://zalando.github.io/restful-api-guidelines/json-guidelines/JsonGuidelines.html#date-property-values-should-conform-to-rfc-3399